tests/models/llava_next_video/test_modeling_llava_next_video.py

```diff
@@ -125,11 +125,12 @@ def __init__(
         self.batch_size = 3
         self.num_channels = 3
         self.image_size = 30
-        self.encoder_seq_length = 127
-        self.image_grid_pinpoints = [[32, 32]]
-        self.num_image_tokens = 88
-        self.num_video_tokens = 32
+
+        self.image_grid_pinpoints = [[16, 16]]
+        self.num_image_tokens = 24
+        self.num_video_tokens = 8
         self.seq_length = seq_length + self.num_image_tokens + self.num_video_tokens
+        self.encoder_seq_length = self.seq_length
```

Comment:
not very related to this PR but lets clean up also, maybe we dont need `encoder-seq-length` in llavas anymore because it was a hack to account for image tokens previously